redshift_sqlalchemy/dialect.py

```diff
@@ -11,6 +11,7 @@
 from sqlalchemy.ext.compiler import compiles
 from sqlalchemy.sql.expression import Executable, ClauseElement
 from sqlalchemy.types import VARCHAR, NullType
+from sqlalchemy.util.compat import string_types
```

Comment:
hmm looking at it it's definitely not public. What would be the best way to go about this? If I were to copy over the code, the linter would complain about unicode or basestring